woocommerce/sensei-share-your-grade
# Add Hungarian translation

Fixes #28.

This adds Hungarian translation files.

## Testing
1. In _Settings_ > _General_, change the _Site Language_ to Hungarian (i.e. Magyar).
2. Go to _Plugins.
3. Disable Sensei LMS.
4. Ensure the notice is translated.
